app/boards/makerbase/mks_canable_v20/mks_canable_v20.conf

```diff
@@ -0,0 +1,20 @@
+# The three lines below are needed to reduce ROM usage
+CONFIG_LTO=y
+CONFIG_SIZE_OPTIMIZATIONS=y
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+
+CONFIG_USB_DEVICE_GS_USB_TIMESTAMP=y
+CONFIG_USB_DEVICE_GS_USB_TERMINATION=n
+CONFIG_USB_DEVICE_GS_USB_MAX_CHANNELS=1
+CONFIG_USB_DEVICE_GS_USB_TIMESTAMP_SOF=y
+CONFIG_USB_DEVICE_GS_USB_IDENTIFICATION=y
+
+CONFIG_LOG=y
+CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
+CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
+CONFIG_USB_DEVICE_GS_USB_LOG_LEVEL_DBG=n
+CONFIG_CANNECTIVITY_LOG_LEVEL_DBG=n
```

Comment:
Please remove these lines. Debug options should not be set on a board-by-board basis.